### ISSUE TITLE
Leaving Lobby After Joining a Lobby

### ISSUE THREAD

**Issue by KaanSacma** [Facepunch/sbox-issues#4492]
### Describe the bug

If someone creates a lobby with the GameNetworkSystem and I try to join it, the Network Session kicks me out because it's already active.

Here is my screenshot from the terminal:
<img width="656" alt="NetworkSessionActiveAlready" src="https://github.com/Facepunch/sbox-issues/assets/91870254/935de9b5-f9bd-406d-ba1d-f6771f39c947">

If we deactivate the Network Session from the scene, we have another issue: the Network is set to a local host.

Here is a screenshot of the issue:
<img width="656" alt="NetworkSessionOff" src="https://github.com/Facepunch/sbox-issues/assets/91870254/51b82e4c-fceb-4736-bbcb-d6e850bd2a52">

Here is a screenshot of the host (he has the same output for each issue):
![LobbyHost](https://github.com/Facepunch/sbox-issues/assets/91870254/ed83bced-fcbc-428b-a883-cc2b83abec77)

### To Reproduce

_For the first Issue_
**Host**
1. Create a Lobby

**Client**
1. Join the Lobby

_For the second issue_
**Scene**
1. Deactivate the Network Session (Host & Client)
2. Save the scene

**Host**
1. Create a Lobby

**Client**
1. Join the Lobby

### Expected behavior

It should not kick and let me connect to the host lobby.

### Media/Files

CreateLobby and JoinLobby (they are in a razor file it's working by pressing a button) function we used:
<img width="466" alt="Screenshot 2024-01-16 at 10 56 23" src="https://github.com/Facepunch/sbox-issues/assets/91870254/55d2a56e-42b8-4e73-ae30-3003a874f88a">

NetworkSession is a component that is used in the scene:
<img width="307" alt="Screenshot 2024-01-16 at 10 58 10" src="https://github.com/Facepunch/sbox-issues/assets/91870254/66962e35-d5af-45b2-92e8-a4bcdee91358">

**Comment by garrynewman** [comment]
In OpenLobby, you're loading a scene, and then connecting to a server - which will load a scene anyway.

Joining a server works fine here so I can't reproduce. If you can get more specific please make a new issue, and make one issue for each problem, rather than compounding them like this (as it's hard to follow, and hard to reproduce)
